Yosemite/Yosemite/Tools/POS/POSOrderService.swift

```diff
@@ -22,7 +22,7 @@ public struct POSOrder {
     public let total: String
     public let totalTax: String
     public let currency: String
-    let items: [POSOrderItem]
+    public let items: [POSOrderItem]
```

Comment:
Note that changing the access control for the items property, as well as the POSOrderItem struct, is only necessary because the following line: `var cleanOrderItems: [POSOrderItem] = order.items.flatMap` needs to access them, so perhaps by exposing that calculation somewhere in the service makes this no longer necessary, and we can keep the property and the struct internal.